src/AWS/Orleans.Streaming.SQS/Streams/SQSStreamBuilder.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Orleans.Configuration;
+using Orleans.Hosting;
+using OrleansAWSUtils.Streams;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Orleans.Streams
+{
+    public class SiloSqsStreamConfigurator: SiloPersistentStreamConfigurator
+    {
+        public SiloSqsStreamConfigurator(string name, ISiloHostBuilder builder)
+            : base(name, builder)
+        {
+            this.siloBuilder
+                .ConfigureApplicationParts(parts => parts.AddFrameworkPart(typeof(SQSAdapterFactory).Assembly))
+                .AddPersistentStreams(name, SQSAdapterFactory.Create)
+                .ConfigureServices(services =>
+                {
+                    services.ConfigureNamedOptionForLogging<SqsOptions>(name)
+                        .ConfigureNamedOptionForLogging<SimpleQueueCacheOptions>(name)
+                        .ConfigureNamedOptionForLogging<HashRingStreamQueueMapperOptions>(name);
+                });
+        }
+
+        public SiloSqsStreamConfigurator ConfigureAzureQueue(Action<OptionsBuilder<SqsOptions>> configureOptions)
+        {
+            this.Configure<SqsOptions>(configureOptions);
+            return this;
+        }
+        public SiloSqsStreamConfigurator ConfigureCacheSize(int cacheSize = SimpleQueueCacheOptions.DEFAULT_CACHE_SIZE)
+        {
+            this.Configure<SimpleQueueCacheOptions>(ob => ob.Configure(options => options.CacheSize = cacheSize));
+            return this;
+        }
+
+        public SiloSqsStreamConfigurator ConfigureQueueMapper(int numOfQueues = HashRingStreamQueueMapperOptions.DEFAULT_NUM_QUEUES)
```

Comment:
ConfigureQueueCount?  Sure, this is configuring the mapper, but users probably don't know what that is.

Reply:
agreed on name QueueMapping